File: 백준/Silver/1436. 영화감독 숌/영화감독 숌.java
import java.io.*;
import java.util.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int N = Integer.parseInt(br.readLine());

        int num = 666;
        int count = 1;

        while (count != N) {
            num++;
            if (String.valueOf(num).contains("666")) {
                count++;
            } // N과 count가 같아질 때 까지 증가시킴 (회차 증가)
        }

        StringBuilder sb = new StringBuilder();
        sb.append(num);
        System.out.print(sb);
    }

}

File: 백준/Silver/1436. 영화감독 숌/영화감독 숌.swift
import Foundation

let n = Int(readLine()!)!

var num: Int = 665
var count: Int = 0

while count < n {
    num += 1
    var temp = num
    
    while temp >= 666 {
        if temp % 1000 == 666 {
            count += 1
            break
        }
        temp /= 10
    }
}

print(num)
